src/etc/inc/gwlb.inc

```diff
@@ -1256,10 +1258,12 @@ function get_gwgroup_members_inner($group, $gateways_status, $gateways_arr, $vip
 	if ($tiers_count == 0) {
 		/* Oh dear, we have no members! Engage Plan B */
 		if (!platform_booting()) {
-			$msg = sprintf(gettext('Gateways status could not be determined, considering all as up/active. (Group: %s)'), $group['name']);
-			log_error($msg);
-			notify_via_growl($msg);
-			//notify_via_smtp($msg);
+			if (isset($config['system']['gw-debug'])) {
```

Comment:
Please move this conditionar to prior `if (!platform_booting() && ...)` and reduce unnecessary indent

Reply:
Done!